.github/workflows/kvrocks.yaml

```diff
@@ -96,3 +96,9 @@ jobs:
           cd build
           cmake ..
           make -j4
+
+  required:
+    runs-on: ubuntu-latest
+    needs: [lint-build-test-on-ubuntu, build-on-macos-latest]
+    steps:
+      - run: echo success
```

Comment:
You may have to check needs.dep in this step. Also you can refer to skywalking's ci yaml file.